src/qibocal/auto/task.py

```diff
@@ -102,7 +101,7 @@ def run(
                 completed.data, completed.data_time = operation.acquisition(
                     parameters, platform=platform
                 )
-        if mode.name in ["autocalibration", "fit"]:
+        if mode & ExecutionMode.FIT:
```

Comment:
```suggestion
        if ExecutionMode.FIT in mode:
```

This is slightly more clear (the `__contains__` is offered by the `Flag`/`IntFlag` implementation, and this is the documented way of doing this check).